sky/packages/workbench/pubspec.yaml

```diff
@@ -6,7 +6,8 @@ homepage: https://github.com/flutter/engine/tree/master/sky/packages/workbench
 dependencies:
   flutter: 
     '0.0.16'
-  sky_tools: any
+  sky_tools:
+    '0.0.34'
```

Comment:
The flutter dependency should take care of this.
If you need this to be pinned as well, though, make sure to update the rev-flutter.dart script.